psydac/api/tests/test_postprocessing.py

```diff
@@ -515,9 +511,9 @@ def test_reconstruct_multipatch():
     A = Square('A',bounds1=bounds1, bounds2=bounds2_A)
     B = Square('B',bounds1=bounds1, bounds2=bounds2_B)
 
-    domain = A.join(B, name='domain',
-                    bnd_minus=A.get_boundary(axis=1, ext=1),
-                    bnd_plus=B.get_boundary(axis=1, ext=-1))
+    connectivity = [((0,1,1),(1,1,-1))]
+    domains = [A,B]
+    domain = Domain.join(domains, connectivity, 'domain')
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)